### PR TITLE
[EN DateTimeV2] Fix for wrongly recognized DateTime entities with "." splitter (#908)

### PR DESCRIPTION
**Pull request by LionbridgeCSII**
Fix for issues reported in #908. 

We did not update the pattern "on 5.11" which is currently parsed as "May 11th" instead of "Nov 5th" (as requested in the issue).   It seems to be a breaking change because there are test cases that rely on the current implementation e.g.:
"Cortana, please set up a Skype call sometime on Friday 7.6 with Jim." -> "July 6th"
"This task should be done on 5.12" -> "May 12th"

Test cases added to DateTimeModel.

### CODE REVIEW

**Review by tellarin — APPROVED** [review]

**Comment by tellarin** [comment]
@LionbridgeCSII Same here. Thanks.

**Comment by LionbridgeCSII** [comment]
> @LionbridgeCSII Same here. Thanks.

resolved.

**Comment by tellarin** [comment]
@LionbridgeCSII Sorry. Simple conflicts again.

File: Specs/DateTime/English/DateTimeModel.json [hunk 1]
```diff
@@ -19968,5 +19968,96 @@
...
+  },
+  {
+    "Input": "I'll go back on 5.1935",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "Results": []
+  },
+  {
+    "Input": "I'll go back 5.11",
```
Comment (tellarin) [inline Specs/DateTime/English/DateTimeModel.json:20008]: Can you add a comment here that this is not supproted as the preposition specifies the interpretation and without it it doesn't make sense?

File: Specs/DateTime/English/DateTimeModel.json [hunk 2]
```diff
@@ -19968,5 +19968,96 @@
...
+              "type": "date",
+              "value": "2017-05-11"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "I'll go back on 5.1935",
```
Comment (tellarin) [inline Specs/DateTime/English/DateTimeModel.json:20001]: Can you add a comment in this one to say that in the future it could be supported as "May-1935"?
Can you also add a spec with some numbers like "0.1993 is not a date" and a spec like "12.12.12.12"?

File: Specs/DateTime/English/DateTimeModel.json [hunk 3]
```diff
@@ -19968,5 +19968,96 @@
...
+  },
+  {
+    "Input": "I'll go back 5.11",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "Results": []
+  },
+  {
+    "Input": "I'll go back during 5.11",
```
Comment (tellarin) [inline Specs/DateTime/English/DateTimeModel.json:20015]: I think we can remove this one.

File: Specs/DateTime/English/DateTimeModel.json [hunk 4]
```diff
@@ -19968,5 +19968,96 @@
...
+              "type": "time",
+              "value": "17:11:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "I'll go back at 5.85",
```
Comment (tellarin) [inline Specs/DateTime/English/DateTimeModel.json:20050]: Please reword this to "The price ended the day at 5.85".

File: Specs/DateTime/English/DateTimeModel.json [hunk 5]
```diff
@@ -19968,5 +19968,96 @@
...
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "Results": []
+  },
+  {
+    "Input": "I'll go back in 5.85",
+    "Context": {
```
Comment (tellarin) [inline Specs/DateTime/English/DateTimeModel.json:20058]: Please, add the same comment as in "5.1935" as in the future this could be interpreted as a date.

**Review by tellarin — CHANGES_REQUESTED** [review]
Sorry, took a closer look at the specs and it would be good to add some info in them.

File: Specs/DateTime/English/DateTimeModel.json (hunk 1 above)
Comment (LionbridgeCSII) [reply Specs/DateTime/English/DateTimeModel.json:20096]: done

File: Specs/DateTime/English/DateTimeModel.json (hunk 2 above)
Comment (LionbridgeCSII) [reply Specs/DateTime/English/DateTimeModel.json:20088]: done and test cases added.

File: Specs/DateTime/English/DateTimeModel.json (hunk 3 above)
Comment (LionbridgeCSII) [reply Specs/DateTime/English/DateTimeModel.json]: removed.

File: Specs/DateTime/English/DateTimeModel.json (hunk 4 above)
Comment (LionbridgeCSII) [reply Specs/DateTime/English/DateTimeModel.json]: done

File: Specs/DateTime/English/DateTimeModel.json (hunk 5 above)
Comment (LionbridgeCSII) [reply Specs/DateTime/English/DateTimeModel.json:20171]: added
